server/services/prismic.js

```diff
@@ -167,3 +165,136 @@ export async function getCuratedList(id: string) {
 
   return curatedList;
 }
+
+async function getExhibitions(page:number = 1, pageSize:number = 40): Promise<any> {
+  const prismic = await getPrismicApi();
+  const exhibitionsList = await prismic.query([
+    Prismic.Predicates.any('document.type', ['exhibitions'])
+  ], { orderings: '[my.exhibitions.start desc]', page, pageSize });
+  return exhibitionsList;
+}
+
+async function createPromos(allResults, type) {
+  switch (type) {
+    case 'exhibition':
+      return await createExhibitionPromos(allResults);
+    case 'event':
+      return await createEventPromos(allResults);
```

Comment:
Given this - should we make sure that we are not showing signs of, "ignorance of the return semantics of async functions." :trollface: 

https://eslint.org/docs/rules/no-return-await